classes/log/store.php

```diff
@@ -63,15 +64,59 @@ protected function is_event_ignored(event_base $event) {
         return $isdisabledevent;
     }
 
+    /**
+     * get eventid from logstore_standard_log
+     * @param event_base $event
+     * @return int
```

Comment:
should probably explain what is returned if/when the id cannot be found

Reply:
0 could identify that the logstore entry is not found. I have added a comment to that effect.